export function errorHandler(err: any, req: any, res: any, next: any) {
  return 'Error Found'
}
